test/saithrift/tests/sail2.py

```diff
@@ -20,6 +20,27 @@
 import sai_base_test
 
 @group('l2')
+class UnknownUnicastTest(sai_base_test.ThriftInterfaceDataPlane):
+    def runTest(self):
```

Comment:
I agree this should be applied in the vlan context, but I suggest to keep it as a separate test since the verification is different.

Another thing to do is to set SAI_SWITCH_ATTR_FDB_UNICAST_MISS_ACTION = DROP.

---
In reply to: [75367140](https://github.com/opencomputeproject/SAI/pull/173#discussion_r75367140) [](ancestors = 75367140)